infra/k8s/install.sh

```diff
@@ -114,8 +114,9 @@ kubectl apply -f ./efs/rbac.yaml \
               -f $EFS_MANIFEST_SPEC
 
 # monitoring and redis.
-echo "installing helm infrastructure"
-helm install testground-infra ./testground-infra
+echo "Installing helm infrastructure"
+helm install --wait --timeout 2m testground-infra ./testground-infra
```

Comment:
While this is fine and would create all the services on the control network (after all `weave` is not installed at that time yet), we have to consider the `healthchecks` and `fixes` and being able to restart/upgrade those services.

Reply:
The only reason why put this up here is weave-service-monitor.yaml. has a CRD which doesn't exist until the operator is installed (the serviceMonitor)

What are we actually waiting for with the timeouts? Does the timeout just ensure the services are created on the right network?

Reply:
The `timeout` and the `wait`, are waiting for the actual resources to be created and running, before we continue.

If we continue to the next step, `weave` gets installed prior to all these resources, and then when we don't have `podAnnotations` set on them, they end up on the `weave` network, which is wrong.